docs/how-to/use-trained-models.txt

```diff
@@ -375,6 +375,6 @@ The checkpoint can then be loaded in Python with this code:
  Next Steps
```

Comment:
Looks like this file has some old checkpoint export APIs embedded in it, can we update those?